src/tests/EdFi.DataManagementService.Tests.E2E/Features/EqualityConstraintValidation.feature

```diff
@@ -1,13 +1,24 @@
 Feature: Equality Constraint Validation
               Equality constraints on the resource describe values that must be equal when posting a resource. An example of an equalityConstraint on bellSchedule:
     "equalityConstraints": [
-        {
-            "sourceJsonPath": "$.classPeriods[*].classPeriodReference.schoolId",
-            "targetJsonPath": "$.schoolReference.schoolId"
-        }
+    {
+    "sourceJsonPath": "$.classPeriods[*].classPeriodReference.schoolId",
+    "targetJsonPath": "$.schoolReference.schoolId"
+    }
     ]
-
+        @ignore
+        # See DMS-259 - Error inserting references
         Scenario: 01 Post a valid bell schedule no equality constraint violations.
+            Given the system has these "descriptors"
+                  | descriptorname                           | codeValue         | description       | namespace                                               | shortDescription  |
+                  | educationOrganizationCategoryDescriptors | School            | School            | uri://ed-fi.org/EducationOrganizationCategoryDescriptor | School            |
+            Given the system has these "schools"
```

Comment:
Please check with some other files, I think we're using different wording. We need to be consistent in all places, so that the step definitions in C# can be reused.

Reply:
I think we need to check the features files (`GIVEN statements ),` there are differences in some files
- Given these schools
- Given the system has these "schools"
- Given the following schools exist